plane/source/docs/basic-setup.rst

```diff
@@ -0,0 +1,299 @@
+
+There are many choices and options for ArduPilot FCs, see `this list of currently supported boards <common-autopilots>`__. 
+Omnibus F4 Pro or Matek F405-Wing are good choices for a basic plane
+configuration. This guide will assume a compareable integrated peripheral setup containing onboard battery monitoring 
+and OSD. Additionally, the following items are required:
+
+-  GPS
+-  RX
+-  ESC & Servos
+-  VTX and Camera
+    
+This guide presumes you are capable to correctly wire all components using the readily available
+resources for the selected board. RCGroups provides forums for hardware help on any of the supported boards.
+Other peripherals (compass, pitot, telemetry, etc.) will not be covered here as they are not necessarily
+required to get you in the air and flying. They can well be added and configured in the future, subject to personal preferences.
+
+
+3. Software
+-----------
+
+This is the part that likely gives potential new users the most concern, that's why this guide
+will focus this.
+
+-  Firmware
+
+Commercially targeted boards like Pixhawk will usually already have some version of ArduPilot loaded, along with an 
+ArduPilot compatible bootloader that will allow updating the firmware using Mission Planner or other GCS software (ground control station) immediately.
+Boards like Omnibus F4 Pro usually don't come with an ArduPilot bootloader by default, so you will need to
+follow `this straight forward procedure to load ArduPilot on your board for the first time <common-loadingfirmware-onto-chibios-only-boards>`__.
+
+-  Mission Planner
+
+Mission Planner (MP) is a ground control station for AP, but also, the primary tool for firmware updates and, most importantly, parameters configuration. MP can be downloaded here:
+http://ardupilot.org/planner/docs/mission-planner-overview.html
+
+There are alternatives,for non-PC users:
+http://ardupilot.org/planner2/index.html
+
+Once the firmware is installed and you are connected to MP or its alternatives, software setup can be begin. 
+Parameters may be changed manually using the Configuration / Tuning Tabs.
+
+Note that some parameters can only be set while dis-armed, and that many parameter groups are not visible until enabled and refreshed, such as the RSSI and OSD parameter groups.
+
+4. Setup
+--------
+
+-  GPS
+
+After connecting to MP, verify that your GPS is recognized and you eventually get a satellite fix. Number of sats and
+HDOP are shown in the Flight Data tab of MP. If no gps is detected, you will see a message in the HUD display stating "No GPS". 
+That means you have hardware debug to do!
+
+Possible issues:
+
+1. Connected to wrong serial port (see your hardware docs for which port AP expects) or incorrectly connected (TX/RX swapped).
+2. Serial port settings were changed from default. ArduPlane expects GPS on Serial 3 at 38.4Kbaud per default. 
+3. Though UBLOX and NMEA-type GPS receivers are supported by ArduPilot these days, your GPS receiver is not supported.
```

Comment:
We support a lot more then these protocols, do you mean "your GPS receiver may not be supported"

Reply:
yes, will modify...thanks